pandas/tests/series/indexing/test_indexing.py

```diff
@@ -853,6 +853,23 @@ def test_setitem_slice_into_readonly_backing_data():
     assert not array.any()
 
 
+def test_access_none_value_in_multiindex():
```

Comment:
can you make a new file in this dir: test_multiindex.py

we have the MI tests elsewhere so ok starting a new file here.

Reply:
should I copy all the imports from test_indexing.py?

Reply:
you need whatever imports make flake happy :->